.github/workflows/manifests-test.yml

```diff
@@ -0,0 +1,21 @@
+name: Test manifests
+on: pull_request
+jobs:
+  regenerate-manifests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Regenerate environment manifests
+        run: |
+          make generate-manifests
+          ERR_MSG="The environment manifests are out of date. Please run 'make generate-manifests' and commit changes!"
+          if ! git diff --exit-code; then
+            echo $ERR_MSG >&2
+            exit 1
+          fi
+          if [ ! -z "$(git status --porcelain)" ]; then
```

Comment:
Not sure if there is any  point in having this second `git status` check?

Reply:
It's needed; the second check verifies whether the make commands don't generate any _new_ files. The first command only verifies that existing files don't get updated.

Reply:
I see, thank you! Here `git status` would catch both tracked and untracked files. I suppose the reason we do the `git diff` is to give a more precise error message. In my PR I have the same message for both, so I could either delete the `git diff` or try to make a more precise error message for it. I would personally prefer to just do `git status` and report an error, without making too many assumptions about what the error is. Would you be ok with that? :)

Reply:
Sure, that's fine. I'll make the change since I'm quickly doing finishing touches so we can merge this now.